ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -81,11 +81,6 @@ const determineView = comparator => {
   return necessaryView
 }
 
-// strip extra quotes
-const stripQuotes = property => {
-  return property.replace(/^"(.+(?="$))"$/, '$1')
-}
```

Comment:
❓ Don't we still need to strip quotes for some values as a stop-gap mitigation for the config admin bug? (@troymohl)

Reply:
I'm not sure this is the case. I changed the type of data structure that was being used here. String -> Object.